setup.py

```diff
@@ -19,7 +19,7 @@
     },
     install_requires=[
         'argparse==1.3.0',
-        'PyGithub==1.25.2',
+        'PyGithub==1.25.2-squash',
```

Comment:
I did a bunch of research and couldnt find this exactly, other than using a requirement.txt